<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="widget_pubquiz_dir">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Sections</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="widget_sectiontable">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>361</width>
      <height>381</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>% score</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="widget_leaderboard">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>90</y>
      <width>461</width>
      <height>381</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Team</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Points</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>60</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Leaderboard</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>530</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Check teams</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_team_problems">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>510</y>
      <width>471</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Team problems</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1007</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>select_dir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widget_sectiontable</sender>
   <signal>cellDoubleClicked(int,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>open_section()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>561</x>
     <y>441</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>select_dir()</slot>
  <slot>open_section()</slot>
 </slots>
</ui>
